lib/backend/k8s/conversion.go

```diff
@@ -445,7 +445,9 @@ func (c Converter) k8sPortToCalico(port extensions.NetworkPolicyPort) []numorstr
 	if port.Port != nil {
 		p, err := numorstring.PortFromString(port.Port.String())
 		if err != nil {
-			log.Panic("Invalid port %+v: %s", port.Port, err)
+			// If we get an invalid port, ignore it.
+			log.Errorf("Invalid port %+v: %s", port.Port, err)
+			return portList
```

Comment:
not related, but curious: why is it returning list at all? it can only have 1 element or be empty

Reply:
hah, yeah good spot.

I think it's a remnant of an earlier version of the code that was missed in a refactor - I believe there used to be a function that could return multiple ports.